play/src/front/Api/ScriptUtils.ts

```diff
@@ -59,21 +63,21 @@ class ScriptUtils {
     }
 
     public startWriting(startWritingEvent: StartWritingEvent, origin?: Window) {
-        const userId = playersStore.addFacticePlayer(startWritingEvent.author || "System");
-
+        //TODO, fix me with new matrix chat integration
```

Comment:
To implement with the new Matrix "writing" code

Reply:
An issue is opened about this: https://github.com/workadventure/workadventure/issues/4033